### DISCUSSION TITLE
技术文章配图指南 · /sketch-and-sketch

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#576] [Announcements]
https://draveness.me/sketch-and-sketch

**Comment by yezuozuo** [comment]
谢谢~

**Comment by mengqingshare** [comment]
非常感谢！我用的比较多的是 processon ，后面试试Sketch~

**Comment by dirtmelon** [comment]
感恩

**Comment by SepCode** [comment]
『你的图片是用什么画的』这句话将为你打上标签，成为你的个人品牌特色之一，挺好😄

**Comment by leejunhui** [comment]
十分感谢~

**Comment by yeahQing** [comment]
所以博主的图片这么好看，使用什么画的?

**Comment by XhinLiang** [comment]
一般来说我喜欢使用 PlantUML，嵌入 Markdown 文档中，可以一并渲染，效果很不错。
但 PlantUML 并不是万能的，在一些比较个性化场景下其语法及其繁琐，可维护性相当低下，在这个时候我会使用 draw.io 绘图。
Sketch 对于我而言学习成本还是太高了..

**Comment by ripperhe** [comment]
很棒

**Comment by xing00lian** [comment]
最近也在使用draw.io绘图，感觉比Sketch学习成本小很多

**Comment by draveness** [comment]
> 最近也在使用draw.io绘图，感觉比Sketch学习成本小很多

之前试过，但是感觉不是特别好看

**Comment by xing00lian** [comment]
> > 最近也在使用draw.io绘图，感觉比Sketch学习成本小很多
> 
> 之前试过，但是感觉不是特别好看

哈哈，楼主可以分享一下使用经验

**Comment by draveness** [comment]
> > > 最近也在使用draw.io绘图，感觉比Sketch学习成本小很多
> > 
> > 
> > 之前试过，但是感觉不是特别好看
> 
> 哈哈，楼主可以分享一下使用经验

分享什么的使用经验啊

**Comment by changtimwu** [comment]
整個 blog 每篇 post 內容都不錯, 請教一下您 blog 是用 wordpress 架設的嗎?

**Comment by draveness** [comment]
> 整個 blog 每篇 post 內容都不錯, 請教一下您 blog 是用 wordpress 架設的嗎?

主要是 Jekyll

**Comment by AlenQi** [comment]
`好的工具也是大家思考的点`，给你点个赞

**Comment by LeoMax-Xiong** [comment]
谢谢博主分享。

**Comment by gaoshuai991** [comment]
想问一下博主的图片是用什么画的？

**Comment by draveness** [comment]
> 想问一下博主的图片是用什么画的？

@gaoshuai991 大哥，这里没写么，这篇文章说的不就是怎么画的么...

**Comment by oldChen89** [comment]
老铁，网站有模板麽？这种样式大爱~ 之前我用jekyll搭过，样式一点也不喜欢~

**Comment by draveness** [comment]
> 老铁，网站有模板麽？这种样式大爱~ 之前我用jekyll搭过，样式一点也不喜欢~

我的模板是私有的，没办法分享，原始的主题是 hugo-lithium 你可以参考下

**Comment by Cherrison** [comment]
https://figmachina.com/articles/learn-about-figma.html
这个也挺不错的 figma, 比  Sketch 更快

**Comment by draveness** [comment]
> https://figmachina.com/articles/learn-about-figma.html
> 这个也挺不错的 figma, 比 Sketch 更快

我确实用了一段时间，下次出个配图指南 2.0

**Comment by dbing** [comment]
优秀哦~ 我辈楷模

**Comment by kergee** [comment]
现在用PPT来画图

**Comment by pdzz** [comment]
感谢优秀的博主，所以你的图是用什么画的

**Comment by minibear2333** [comment]
我发现图片点击无法查看原图

**Comment by luckyray-fan** [comment]
sketch 组件库可以整一个吗, 挺喜欢你的配色

**Comment by Mark24Code** [comment]
感谢楼主

**Comment by originalix** [comment]
很棒的文章，所以博主的图是用什么画的？

**Comment by szupzj18** [comment]
作者的文章写得真的很不错，简洁美观的图片更是为深刻的思考锦上添花。👍

**Comment by weihong363** [comment]
感谢博主

**Comment by ting-talk** [comment]
感谢你的写作，已为你转发。

**Comment by jiekun** [comment]
博主的图片这么好看，使用什么画的?

**Comment by loo2k** [comment]
这里再推荐下可以用 Figma 比 Sketch 要方便很多，重点是跨平台、个人用户免费使用。

**Comment by Utachen** [comment]
博主配图真好看，所以是用什么画的？

**Comment by YunLingCun** [comment]
使用keynote 也可以做出这些图来

**Comment by JiapengLi** [comment]
inkcape也是一个不错的开源工具

**Comment by zachzhao1984** [comment]
推荐可以试试 excalidraw https://excalidraw.com/

**Comment by mengqi92** [comment]
分享的工具很有用，关于设计系统和规则体系也很有启发，点个赞！

有一个关于配色的建议：
从可用性的角度，WCAG 2.0 关于色彩对比度要求图形前景/背景对比度至少要达到 3:1（[Color contrast - Accessibility - MDN Web Docs
](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)），我测试了一下，白色前景文字搭配浅色背景的图片，对比度只有 1.4~2:1，都不满足 WCAG 要求（macOS 可以用 [Color Contrast Analyser](https://www.tpgi.com/color-contrast-checker/) 测试，Windows 可以用 [Accessibility Insights for Windows](https://go.microsoft.com/fwlink/?linkid=2075365) 测试）。实际肉眼观测，也是能感到有一点分辨困难的，希望作者还能改进一下，谢谢！

**Comment by draveness** [comment]
> @mengqi92
> 分享的工具很有用，关于设计系统和规则体系也很有启发，点个赞！
> 
> 有一个关于配色的建议：
> 从可用性的角度，WCAG 2.0 关于色彩对比度要求图形前景/背景对比度至少要达到 3:1（[Color contrast - Accessibility - MDN Web Docs
> ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)），我测试了一下，白色前景文字搭配浅色背景的图片，对比度只有 1.4~2:1，都不满足 WCAG 要求（macOS 可以用 [Color Contrast Analyser](https://www.tpgi.com/color-contrast-checker/) 测试，Windows 可以用 [Accessibility Insights for Windows](https://go.microsoft.com/fwlink/?linkid=2075365) 测试）。实际肉眼观测，也是能感到有一点分辨困难的，希望作者还能改进一下，谢谢！
> 

高级啊，学习了

**Comment by XhinLiang** [comment]
> 画图的工具并不重要，重要的其实是你应该如何形成自己的规则体系

不太同意，规则体系是建立在画图工具上的。而且基于本文的内容，也是以画图工具作为分类的。不同的画图工具，使用习惯有很大差异，思路可以复用，但是还是需要学习成本的。

**Comment by szupzj18** [comment]
> > 画图的工具并不重要，重要的其实是你应该如何形成自己的规则体系
> 
> 不太同意，规则体系是建立在画图工具上的。而且基于本文的内容，也是以画图工具作为分类的。不同的画图工具，使用习惯有很大差异，思路可以复用，但是还是需要学习成本的。

我倒是觉得应该是按照自己的需要去选择画图工具，而不是先有工具再有体系。

**Comment by originalix** [comment]
> 推荐可以试试 excalidraw https://excalidraw.com/

工具风格不错，赞一下

**Comment by lihuadong** [comment]
深表认同，读博主的文章，不管是内容质量还是排版。祝好~

**Comment by nonacosa** [comment]
> @nouh
> 推荐可以试试 excalidraw https://excalidraw.com/

帅

**Comment by zzsakurazz** [comment]
非常感谢

**Comment by feizeikesi** [comment]
受教了

**Comment by wfnuser** [comment]
tql

**Comment by lovebingheji** [comment]
非常感谢

**Comment by Megrax** [comment]
量产手绘风格可以试试：https://app.excalidraw.com/
